oauth2/oauth2-resource-server/src/test/java/org/springframework/security/oauth2/server/resource/authentication/OAuth2IntrospectionAuthenticationTokenTests.java

```diff
@@ -46,15 +46,13 @@
 	private final OAuth2AccessToken token =
 			new OAuth2AccessToken(OAuth2AccessToken.TokenType.BEARER,
 				"token", Instant.now(), Instant.now().plusSeconds(3600));
-	private final Map<String, Object> attributes = new HashMap<>();
 	private final String name = "sub";
-
-	@Before
-	public void setUp() {
-		this.attributes.put(SUBJECT, this.name);
-		this.attributes.put(CLIENT_ID, "client_id");
-		this.attributes.put(USERNAME, "username");
-	}
+	private Map<String, Object> attributesMap = new HashMap<String, Object>() {{
```

Comment:
While this is a clever construct for initializing a `Map`, it's not used across the codebase. So, just for consistency, let's please still use the `setUp` method.

Reply:
Sounds good, I didn't know @Before would allow `setUp` to populate the `attributesMap` before the OAuth2TokenAttributes `attributes` was made.